source/js/dismissableNotices.ts

```diff
@@ -0,0 +1,113 @@
+enum NoticeTimeout {
+  Session = 'session',
+  Permanent = 'permanent',
+  Imidiate = 'imidiate',
```

Comment:
This is a typo i think. Its supposed to be "immediate". Needs to be changed in all occurences